Makefile

```diff
@@ -602,12 +602,20 @@ docker-build-all: $(addprefix docker-build-,$(ALL_ARCH)) ## Build docker images
 docker-build-%:
 	$(MAKE) ARCH=$* docker-build
 
-ALL_DOCKER_BUILD = core kubeadm-bootstrap kubeadm-control-plane clusterctl
+ALL_DOCKER_BUILD = core kubeadm-bootstrap kubeadm-control-plane capd test-extension clusterctl
 
 .PHONY: docker-build
-docker-build: docker-pull-prerequisites ## Run docker-build-* targets for all providers
+docker-build: docker-pull-prerequisites ## Run docker-build-* targets for all the images
 	$(MAKE) ARCH=$(ARCH) $(addprefix docker-build-,$(ALL_DOCKER_BUILD))
 
+ALL_DOCKER_BUILD_E2E = core kubeadm-bootstrap kubeadm-control-plane capd test-extension
+
+.PHONY: docker-build-e2e ## Run docker-build-* targets for all the images required by E2E tests
```

Comment:
The comment on L613 isn't shown in the Makefile help and is very similar to the one below that is shown.

Reply:
```
$ make
(...)
build:
 (...)
  docker-build-all                               Build docker images for all architectures
  docker-build                                   Run docker-build-* targets for all the images
  docker-build-e2e                               Run docker-build-* targets for all the images with settings to be used for the e2e tests
```